library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/BodyLineItemCreator.kt

```diff
@@ -0,0 +1,46 @@
+package com.chuckerteam.chucker.internal.ui.transaction
+
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.ClickableSpan
+import android.view.View
+import com.chuckerteam.chucker.internal.support.Logger
+
+internal class BodyLineItemCreator(private val bodyLine: String, private val onUrlClicked: (String) -> Unit) {
+
+    fun create(): SpannableStringBuilder =
+        SpannableStringBuilder().apply {
```

Comment:
You can use `buildSpannedString()` instead.

Reply:
That returns `SpannedString` and `BodyLineItem` accepts `SpannableStringBuilder`. (`TransactionBodyAdapter` uses `SpannableStringBuilder#clearSpans`)

Reply:
Ah ok, Never mind then. :)